stdlib/test/utils/test_tuple.mojo

```diff
@@ -15,6 +15,7 @@
 from testing import assert_equal, assert_false, assert_true
 
 from utils import StaticTuple, StaticIntTuple
+from utils.static_tuple import InlineArray
```

Comment:
**Suggestion** Instead of this, modify `utils/__init__.mojo` to public export `InlineArray`.  Then, you can just modify the line above here to import `InlineArray`.